src/main/java/frc/robot/commands/teleopCanal.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.CanalSubsystem;
+
+public class teleopCanal extends CommandBase {
+
+  private CanalSubsystem canal;
+  /** Creates a new teleopCanal. */
+  public teleopCanal(CanalSubsystem canalArgs) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    this.canal = canalArgs;
+
+    addRequirements(canal);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    // canal.setSpeedBack(-Constants.BELT_SPEED);
```

Comment:
Why are all the canal functions commented out?

Reply:
"angry" -ryan